docker/build.sh

```diff
@@ -169,10 +175,16 @@ else
                                    --build-arg "CMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}"      \
                                    --build-arg "BUILD_TEST=${BUILD_TEST}"                  \
                                    --build-arg "BUILD_BENCHMARK=${BUILD_BENCHMARK}"        \
+                                   --build-arg "BUILD_DALI_KERNELS=${BUILD_DALI_KERNELS}"  \
+                                   --build-arg "BUILD_DALI_PIPELINE=${BUILD_DALI_PIPELINE}"\
+                                   --build-arg "BUILD_DALI_OPERATORS=${BUILD_DALI_OPERATORS}"\
                                    --build-arg "BUILD_NVTX=${BUILD_NVTX}"                  \
                                    --build-arg "BUILD_PYTHON=${BUILD_PYTHON}"              \
                                    --build-arg "BUILD_LMDB=${BUILD_LMDB}"                  \
                                    --build-arg "BUILD_JPEG_TURBO=${BUILD_JPEG_TURBO}"      \
+                                   --build-arg "BUILD_FFMPEG=${BUILD_FFMPEG}"              \
```

Comment:
remove, BUILD_FFMPEG is an internal variable (BUILD_NVDEC is the exposed option)

Reply:
Done